lib/hunger_games/schedule_requests_solver/simple_schedule_request_solver.ex

```diff
@@ -0,0 +1,125 @@
+defmodule HungerGames.SimpleScheduleRequestSolver do
+  @moduledoc """
+  This is a simplest possible ScheduleRequestSolver.
+
+  It is based on time of requests and does not feature collision detection
+  """
+  @behaviour HungerGames.ScheduleRequestSolver
+
+  alias HungerGames.{
+    ClassRequests.ClassRequest,
+    Requests.Request,
+    ScheduleRequestSolver.Class,
+    Schedules.Schedule
+  }
+
+  @impl true
+  def solve_schedule(%Schedule{classes: classes, requests: requests}) do
+    classes =
+      classes
+      |> Enum.map(&Map.from_struct(&1))
+      |> Enum.map(&struct(Class, &1))
```

Comment:
what was your intention here? struct to map and back to struct?

Reply:
From database class, to ScheduleRequestSolver Class to limit to filter out unnecessary fields and add some others